graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/Message.java

```diff
@@ -253,6 +253,9 @@ public void addField(final String key, final Object value) {
             if (LOG.isDebugEnabled()) {
                 LOG.debug("Ignoring invalid or reserved key {} for message {}", key, getId());
             }
+            if (key != null && key.contains(".")) {
+                LOG.warn("Ignoring field \"{}\"=\"{}\" - Keys with a \".\" character are not supported anymore since Graylog 2.0! (Message[{}])", key, value, toString());
```

Comment:
I think this shouldn't print the complete message but only the message Id for reference. Otherwise the log statements might become quite big and unwieldy.